indico/migrations/versions/20220630_1206_7c99d8f13df7_add_translation_of_title_and_.py

```diff
@@ -0,0 +1,32 @@
+"""Add translation of title and description to categories
+
+Revision ID: 7c99d8f13df7
+Revises: 0c4bb2973536
+Create Date: 2022-06-30 12:06:05.586959
+"""
+
+import sqlalchemy as sa
+from alembic import op
+from sqlalchemy.dialects import postgresql
+
+
+# revision identifiers, used by Alembic.
+revision = '7c99d8f13df7'
+down_revision = '0c4bb2973536'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.add_column('categories', sa.Column('title_translations', postgresql.JSONB(astext_type=sa.Text()), nullable=True), schema='categories')
+    op.execute(r"UPDATE categories.categories SET title_translations = '{}'::jsonb")
```

Comment:
Create the column with a server_default and remove it again afterwards. See e.g. `20200623_1718_05f227f4b938_add_metadata_to_event_logs.py` for an example.

Reply:
Thanks for the example!